modules/mongodb/charts.go

```diff
@@ -0,0 +1,490 @@
+		{ID: "asserts.user", Name: "user", Algo: module.Incremental},
+		{ID: "asserts.tripwire", Name: "tripwire", Algo: module.Incremental},
+		{ID: "asserts.rollovers", Name: "rollovers", Algo: module.Incremental},
+	},
+}
+
+// option charts
+var chartTransactionsCurrent = module.Chart{
+	ID:    "transactionsCurrent",
+	Title: "Current Transactions",
+	Units: "transactions",
+	Fam:   "transactionsCurrent",
+	Ctx:   "mongodb.transactionsCurrent",
+	Dims: module.Dims{
+		{ID: "transactions.currentActive", Name: "current active"},
+		{ID: "transactions.currentInactive", Name: "current inactive"},
+		{ID: "transactions.currentOpen", Name: "current open"},
+		{ID: "transactions.currentPrepared", Name: "current prepared"},
+	},
+}
+
+var chartGlobalLockActiveClients = module.Chart{
+	ID:    "globalLockActiveClients",
+	Title: "Active Clients",
+	Units: "clients",
+	Fam:   "clients",
+	Ctx:   "mongodb.currentQueue",
+	Dims: module.Dims{
+		{ID: "globalLock.activeClients.readers", Name: "readers"},
+		{ID: "globalLock.activeClients.writers", Name: "writers"},
+	},
+}
+
+var chartCollections = module.Chart{
+	ID:    "catalogStats",
+	Title: "Catalog Stats",
+	Units: "objects",
+	Fam:   "catalogStats",
+	Ctx:   "mongodb.catalogStats",
+	Type:  module.Stacked,
+	Dims: module.Dims{
+		{ID: "catalogStats.collections", Name: "collections"},
+		{ID: "catalogStats.capped", Name: "capped"},
+		{ID: "catalogStats.timeseries", Name: "timeseries"},
+		{ID: "catalogStats.views", Name: "views"},
+		{ID: "catalogStats.internalCollections", Name: "internalCollections"},
+		{ID: "catalogStats.internalViews", Name: "internalViews"},
+	},
+}
+
+var (
+	chartTcmallocGeneric = module.Chart{
+		ID:    "tcmallocGeneric",
+		Title: "Tcmalloc generic metrics",
+		Units: "MiB",
+		Fam:   "tcmalloc",
+		Ctx:   "mongodb.tcmallocGeneric",
+		Dims: module.Dims{
+			{ID: "tcmalloc.generic.current_allocated_bytes", Name: "current_allocated_bytes", Div: 1 << 20},
+			{ID: "tcmalloc.generic.heap_size", Name: "heap_size", Div: 1 << 20},
+		},
+	}
+	chartTcmalloc = module.Chart{
+		ID:    "tcmalloc",
+		Title: "Tcmalloc",
+		Units: "KiB",
+		Fam:   "tcmalloc",
+		Ctx:   "mongodb.tcmalloc",
+		Dims: module.Dims{
+			{ID: "tcmalloc.tcmalloc.pageheap_free_bytes", Name: "Pageheap free", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_unmapped_bytes", Name: "Pageheap unmapped ", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.current_total_thread_cache_bytes", Name: "Total threaded cache", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.total_free_bytes", Name: "Free", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_committed_bytes", Name: "Pageheap committed", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_total_commit_bytes", Name: "Pageheap total commit", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_total_decommit_bytes", Name: "Pageheap decommit", Div: 1024},
+			{ID: "tcmalloc.tcmalloc.pageheap_total_reserve_bytes", Name: "Pageheap reserve", Div: 1024},
+		},
```

Comment:
```suggestion
	chartTcmalloc = module.Chart{
		ID:    "tcmalloc",
		Title: "Tcmalloc",
		Units: "bytes",
		Fam:   "memory",
		Ctx:   "mongodb.tcmalloc",
		Dims: module.Dims{
			{ID: "tcmalloc.tcmalloc.pageheap_free_bytes", Name: "Pageheap free"},
			{ID: "tcmalloc.tcmalloc.pageheap_unmapped_bytes", Name: "Pageheap unmapped "},
			{ID: "tcmalloc.tcmalloc.current_total_thread_cache_bytes", Name: "Total threaded cache"},
			{ID: "tcmalloc.tcmalloc.total_free_bytes", Name: "Free"},
			{ID: "tcmalloc.tcmalloc.pageheap_committed_bytes", Name: "Pageheap committed"},
			{ID: "tcmalloc.tcmalloc.pageheap_total_commit_bytes", Name: "Pageheap total commit"},
			{ID: "tcmalloc.tcmalloc.pageheap_total_decommit_bytes", Name: "Pageheap decommit"},
			{ID: "tcmalloc.tcmalloc.pageheap_total_reserve_bytes", Name: "Pageheap reserve"},
		},
```

- didn't find these metrics in [serverStatus](https://docs.mongodb.com/manual/reference/command/serverStatus/)
- I guess it makes sense to move it to "memory"
- it is in bytes, correct? If so there is no need to convert to something else

Reply:
correct, they are in bytes